gremlin-go/docker-compose.yml

```diff
@@ -37,6 +39,7 @@ services:
       - .:/go_app
       - ../gremlin-test/features:/gremlin-test
     environment:
+      - HOST_SERVER=gremlin-test-server
       - CUCUMBER_FEATURE_FOLDER=/gremlin-test
       - GREMLIN_SERVER_URL=ws://gremlin-test-server:45940/gremlin
       - GREMLIN_SERVER_BASIC_AUTH_URL=wss://gremlin-test-server:45941/gremlin
```

Comment:
out of curiosity do you envision any issues with getting the kerberos version of the server running under this model?

Reply:
I cannot say for certain as I have not tried setting up a server with kerberos nor am I familiar enough with kerberos, but from what I understand, we can install kerberos to docker and containerize the KDC, server and client, exposing ports should allow communications between them in docker compose (I imagine issues, if any, would arise here). Do you have any particular concerns we should keep in mind?